### ISSUE TITLE
The result of SintelTrainingFinalValid

### ISSUE THREAD

**Issue by Ecalpal** [visinf/irr#9]
HI!

I ran IRR-PWC_sintel.sh three times, and give the result as follows.

```
2019-09-05 11:27:10 ==> Epoch 1/1
2019-09-05 11:27:10   lr: 0.001
==> Validate: 100%|######################################|  34/34  00:36<00:00  0.92it/s  F1_avg=0.7416, epe_avg=2.7496  
==> Progress: 100%|##################################################|  1/1  00:36<00:00 36.88s/ep  best_epe_avg=2.7496 
```

```
2019-09-05 11:28:22 ==> Epoch 1/1
2019-09-05 11:28:22   lr: 0.001
==> Validate: 100%|######################################|  34/34  00:36<00:00  0.94it/s  F1_avg=0.7416, epe_avg=2.7451  
==> Progress: 100%|##################################################|  1/1  00:36<00:00 36.21s/ep  best_epe_avg=2.7451 
```

```
2019-09-05 11:33:18 ==> Epoch 1/1
2019-09-05 11:33:18   lr: 0.001
==> Validate: 100%|######################################|  34/34  00:36<00:00  0.93it/s  F1_avg=0.7417, epe_avg=2.7598  
==> Progress: 100%|##################################################|  1/1  00:36<00:00 36.53s/ep  best_epe_avg=2.7598
```
In Table 5 of the paper, the result of IRR-PWC(Ours) in Sintel Training is 2.51
Why is it different from the result of the paper?

Or saved_check_point/pwcnet/IRR-PWC_sintel/checkpoint_latest.ckpt is not the weight which uploaded in sintel?

**Comment by hurjunhwa** [comment]
Hi,

The 2.51 in Table 5 of the main paper is the EPE evaluated on the whole Sintel Final Training images, not only on the validation set.

In the script file, if you use ``--validation_dataset=SintelTrainingFinalFull `` instead of ``--validation_dataset=SintelTrainingFinalValid `` , you would get the number :)

**Comment by Ecalpal** [comment]
Yes, I got it! 
Sorry for my carelessness.

```
2019-09-05 16:41:45 ==> Epoch 1/1
2019-09-05 16:41:45   lr: 0.001
==> Validate: 100%|###########################################|  261/261  03:57<00:00  1.10it/s  F1_avg=0.7044, epe_avg=2.4999  
==> Progress: 100%|########################################################|  1/1  03:57<00:00 237.14s/ep  best_epe_avg=2.4999 
```
